flow/project.py

```diff
@@ -5006,7 +5014,10 @@ class MyProject(FlowProject):
             "for the majority of runtime required for status determination. "
             "Optionally provide a filename pattern to select for what files "
             "to show result for. Defaults to the main module. "
-            "(requires pprofile)",
+            "(requires profile)",
```

Comment:
This is not a typo. The package we import is called `pprofile`. https://github.com/glotzerlab/signac-flow/blob/38a7b8b425cf4a1a1d4faa73a3e1b8d2dc6573e3/flow/project.py#L2904

```suggestion
            "(requires pprofile)",
```